polaris-gateway/Handlers/InitailizationHandler.cs

```diff
@@ -1,67 +0,0 @@
-using Common.Telemetry;
-using Microsoft.AspNetCore.Http;
-using PolarisGateway.Validators;
-using Common.Extensions;
-using Common.Constants;
-using PolarisGateway.Exceptions;
-using PolarisGateway.Extensions;
-
-namespace PolarisGateway.Handlers;
-
-public class InitializationHandler : IInitializationHandler
```

Comment:
Why this handler has been removed?

Reply:
It has been removed as part of the original upgrade commit, therefore assume it was unused an/or moved as part of these changes.